plugins/org.eclipse.xtext/src/org/eclipse/xtext/generator/URIBasedFileSystemAccess.xtend

```diff
@@ -62,7 +63,7 @@ class URIBasedFileSystemAccess extends AbstractFileSystemAccess2 {
 		val outlet = pathes.get(outputConfiguration)
 		if (outlet == null)
 			throw new IllegalArgumentException("A slot with name '" + outputConfiguration + "' has not been configured.");
-		val uri = URI.createURI(outlet + "/" + path)
+		val uri = URI.createFileURI(outlet + File.separator + path)
```

Comment:
Doesn't this turn it back into a JavaIOFileSystemAccess? I thought this was meant to abstract over file:// and temp:// in IDEA for instance.